gcp-common/build.gradle

```diff
@@ -3,12 +3,11 @@ plugins {
 }
 
 dependencies {
-    compileOnly(mn.micronaut.runtime)
-
+    compileOnly(mn.micronaut.discovery.core)
     implementation(mn.micronaut.jackson.databind)
```

Comment:
why do we still depend on databind here?

Reply:
https://github.com/micronaut-projects/micronaut-gcp/issues/842